rxandroid-framework/src/test/java/rx/android/app/support/ReactiveDialogTest.java

```diff
@@ -0,0 +1,116 @@
+package rx.android.app.support;
+
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentManager;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Mock;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.shadows.ShadowDialog;
+import rx.Observable;
+import rx.Observer;
+import rx.android.app.ReactiveDialogResult;
+
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertNotNull;
```

Comment:
You have an explicit import and a star import right above it. Please avoid star imports.